nnpdf_data/nnpdf_data/commondata/ATLAS_SINGLETOP_7TEV/metadata.yaml

```diff
@@ -60,47 +60,47 @@ implemented_observables:
       - ATLAS_SINGLETOP_TCH_DIFF_7TEV_TBAR_RAP_NORM-aMCfast_obs_1
       - ATLAS_SINGLETOP_TCH_DIFF_7TEV_TBAR_RAP_NORM-aMCfast_obs_2
     - - ATLAS_SINGLETOP_TCH_DIFF_7TEV_TBAR_RAP_NORM-ATLAS_SINGLETOP_TCH_R_7TEV_TB
-  data_uncertainties: []
+  data_uncertainties: [uncertainties_TBAR-Y-NORM.yaml]
   variants:
     legacy:
       data_uncertainties:
       - uncertainties_legacy_TBAR-Y-NORM.yaml
-  data_central: data_legacy_TBAR-Y-NORM.yaml
+  data_central: data_TBAR-Y-NORM.yaml
   ported_from: ATLAS_SINGLETOP_TCH_DIFF_7TEV_TBAR_RAP_NORM
 - observable_name: T-Y-NORM
   observable:
     description: Heavy Quarks Production Single Quark Rapidity Distribution
     label: ATLAS single top $y_t$ (normalised)
     units: ''
   process_type: HQP_YQ
-  tables: []
+  tables: [17, 26, 34]
   npoints: []
   ndata: 3
   plotting:
-    kinematics_override: hqp_yq_sqrt_scale
+    kinematics_override: identity
     dataset_label: ATLAS single top $y_t$ (normalised)
     y_label: $(1/\sigma_t)d\sigma_t/d|y_t|$
     figure_by:
-    - k3
-    plot_x: k1
+    - sqrts
```

Comment:
With respect to k1/k2/k3, this comment I left in #2178 applies here as well: https://github.com/NNPDF/nnpdf/pull/2178#pullrequestreview-2409495875 basically, look in process option and use one of the process there. Since (for single top) there will be no process I believe, you need to create it.

That means that you also need to add a function to define the xq coverage plot (and you are free to use as input variables what you want as long as you get the x-q plot as the output).

The best way to check is to do a data-theory comparison plot and a x-q coverage plot `{@plot_xq2@}` in master and this branch and seeing that they look the same.